.coveragerc

```
@@ -4,7 +4,7 @@ skip_covered = false
 
 [paths]
 source =
```

I was given feedback that this name is confusing (or `src` that I was using in some projects) so I'm now migrating my projects to something a bit more verbose in hopes of preventing people from making assumptions…
```suggestion
_site-packages-to-src-mapping =
```